src/App.tsx

```diff
@@ -15,48 +16,116 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  BY_NONE = 0,
```

Comment:
0 is default value for first element of enum
```suggestion
  BY_NONE,
```